externs/shaka/player.js

```diff
@@ -129,6 +131,11 @@ shakaExtern.Stats;
  *   (only for video tracks) The width of the track in pixels.
  * @property {?number} height
  *   (only for video tracks) The height of the track in pixels.
+ * @property {?number} frameRate
+ *   (only for video tracks) The video framerate provided in the manifest.
```

Comment:
This information is not required or used in streaming or setup, so how about"provided in the manifest, if present."